glean/src/core/context.ts

```diff
@@ -45,6 +47,13 @@ export class Context {
   private _initialized = false;
   // Whether or not Glean is in testing mode.
   private _testing = false;
+  // A map of metric types and their constructors.
+  // This map is dinamically filled everytime a metric type is constructed.
```

Comment:
```suggestion
  // This map is dynamically filled everytime a metric type is constructed.
```

(should be changed in the commit message too)